src/foraging_gui/Dialogs.py

```diff
@@ -1343,8 +1337,7 @@ def _StartCamera(self):
             return
         if self.StartRecording.isChecked():
             self.StartRecording.setStyleSheet("background-color : green;")
-            self.WarningLabelCameraOn.setText('Camera is turning on')
-            self.WarningLabelCameraOn.setStyleSheet(self.MainWindow.default_warning_color)
```

Comment:
![image](https://github.com/user-attachments/assets/222c04b6-20f7-4741-aa2b-3aec0fd9fbf0)

Reply:
Again, I would duplicate this one